cle/backends/java/apk.py

```diff
@@ -53,15 +60,11 @@ def __init__(self, apk_path, binary_stream, entry_point=None, entry_point_params
         else:
             l.info("Using user defined JNI lib(s) %s (load path(s) %s)", jni_libs, jni_libs_ld_path)
 
-        if not entry_point:
-            try:
-                from pyaxmlparser import APK as APKParser
-                apk_parser = APKParser(apk_path)
-                main_activity = apk_parser.get_main_activity()
-                entry_point = main_activity + '.' + 'onCreate'
-                entry_point_params = ('android.os.Bundle',)
-            except ImportError:
-                l.error("Install pyaxmlparser to identify APK entry point.")
+        if not entry_point and PYAXMLPARSER_INSTALLED:
+            apk_parser = APKParser(apk_path)
+            main_activity = apk_parser.get_main_activity()
+            entry_point = main_activity + '.' + 'onCreate'
+            entry_point_params = ('android.os.Bundle',)
```

Comment:
Can we handle the case where pyaxmlparser is not installed?

Reply:
I handled to check the condition pyaxmlparser installed, but an issue remained you commented after.